tableconvert/src/tableconvert.c

```diff
@@ -35,6 +35,19 @@ PLUGIN_SET_TRANSLATABLE_INFO(
 GeanyPlugin	 	*geany_plugin;
 GeanyData	   	*geany_data;
 
+
+PluginCallback plugin_callbacks[] =
+{
+	{ "editor-notify", (GCallback) &cb_table_convert_change_document, FALSE, NULL },
+	{ "document-activate", (GCallback) &cb_table_convert_change_document, FALSE, NULL },
+	{ "document-filetype-set", (GCallback) &cb_table_convert_change_document, FALSE, NULL },
+	{ "document-new", (GCallback) &cb_table_convert_change_document, FALSE, NULL},
+	{ "geany-startup-complete", (GCallback) &cb_table_convert_change_document, FALSE, NULL },
+	{ "document-close", (GCallback) &cb_table_convert_change_document, FALSE, NULL},
+	{ NULL, NULL, FALSE, NULL }
+};
```

Comment:
Do you really need all of this callbacks? I guess at least ```geany-startup-complete``` is not required or am I overseeing something?

Reply:
I guess that's fine now.